alphadia/workflow/optimization.py

```diff
@@ -226,9 +230,10 @@ def _propose_new_parameter(self, df):
         float
             The proposed new value for the search parameter.
 
-
         """
-        pass
+        return self.update_factor * self.workflow.calibration_manager.get_estimator(
+            self.estimator_group_name, self.estimator_name
+        ).ci(df, self.update_interval)
```

Comment:
when reading `update_interval` I think of `time between updates` .. mb rename to `update_confidence_interval`?

Reply:
I am wondering if we could find some better term than "confidence interval", because it isn't really a confidence interval in the normal sense. Perhaps ``update_percentile_range``?